### ISSUE TITLE
[Annoyance] Pull secret always asked during on boarding process

### ISSUE THREAD

**Issue by adrianriobo** [crc-org/tray-electron#106]
Testing an upgrade scenario where:

* I did not run `crc cleanup`  (as I want to keep pull secret on the keyring)
* Remove crc home folder

After install crc version, the onboarding process asks me for the pull secret (the pull secret is already stored at the keyring)

**Comment by gbraad** [comment]
We can NOT check for the existence of the pull-secret, as this requires the daemon.
The pull-secret is 'optional' to be provided during the onboarding.

The actual check will happen when the daemon is available.

**Comment by adrianriobo** [comment]
Yeah, I am fine with the daemon as single point of truth (or state :)) , just point out the case, as this is a technical constraint but from an UX perspective is a bit strange (at least).

**Comment by gbraad** [comment]
Annoyance rather than a blocker. The field is optional (does not prevent the user from progressing).
And the onboarding is a single-time event, unless the user 'resets'.

Parking on backlog
